client/state/sites/selectors.js

```diff
@@ -142,6 +143,26 @@ export function getSitePlan( state, siteId ) {
 		return null;
 	}
 
+	if ( get( site.plan, 'expired', false ) ) {
+		if ( site.jetpack ) {
+			return {
+				product_id: 2002,
+				product_slug: 'jetpack_free',
+				product_name_short: 'Free',
+				free_trial: false,
+				expired: false
+			};
+		}
+
+		return {
```

Comment:
When your plan expires, the site goes back to free functionality since you haven't renewed. The subscription is cleaned up eventually. This particular selector is abstracts out needing to check for expired values. If we end up in a situation where we need to know something is recently expired, we can add another selector.